### DISCUSSION TITLE
Help Needed regarding Github and the course.

### DISCUSSION THREAD

**Discussion by harishchaurasia** [microsoft/ML-For-Beginners#162] [Q&A] [answered]
Hey there,
I am a 12th-grade student from India. I graduated High-School this year.
I am totally new to GitHub, and nothing is making sense to me.
I wanna do this course but I'm not sure how to get started and what to do.
Please somebody help me.

**Comment by jlooper** [comment, marked as answer]
hi, did you read the home README.md file? There's a section there on how to get started. What you should do is fork this repo to your own account and then work through the 24 lessons at your own pace. You can run the code locally by making a copy of your forked repo to your local computer, opening it in Visual Studio Code, and reading through the lessons. You can run notebooks right in VS Code.

**Reply by harishchaurasia** [reply]
Thanks and I also gotta know If I would also receive a certificate for this course?
And how do I fork the repo? Actually, I signed up here just yesterday so I'm totally dumb rn.
Btw thanks for the help...I'll let you know If I need more help.

**Reply by jlooper** [reply]
Hi, there’s no certificate as it’s a self-study course. You might take this lesson for help with GitHub https://github.com/microsoft/Web-Dev-For-Beginners/blob/main/1-getting-started-lessons/2-github-basics/README.md
